app/repositories/UploadJourneyRepository.scala

```diff
@@ -101,7 +106,7 @@ class UploadJourneyRepository @Inject()(
           .map {
```

Comment:
Could you change this line to retrieve a SensitiveUploadJourney:
https://github.com/hmrc/penalties-appeals-frontend/pull/509/files#diff-8b51f0a8b26c714ccbf85b6e9588b7c2253b4bcc904928b933d067434849201fL80